widgetastic_manageiq/__init__.py

```diff
@@ -5730,12 +5732,12 @@ def value(self):
     def read(self):
         return self.value
 
-    def fill(self, value):
+    def fill(self, value, include_domain=False):
```

Comment:
could you explain what it is for ?

Reply:
- We need to include domain while selecting entry point in catalog item.
- This is useful if we have same tree but different domains available. So in this case we need to select domain prefix.


![screen_cfme](https://user-images.githubusercontent.com/25399590/62133794-0dd45500-b2fd-11e9-9ffe-91b04d9bcaba.png)